src/databricks/labs/ucx/config.py

```diff
@@ -198,36 +198,36 @@ def to_account_client(self) -> AccountClient:
 
 @dataclass
 class WorkspaceConfig(_Config["WorkspaceConfig"]):
-    inventory_database: str
-    groups: GroupsConfig
-    instance_pool_id: str = None
-    warehouse_id: str = None
-    connect: ConnectConfig | None = None
-    num_threads: int | None = 10
-    log_level: str | None = "INFO"
+    connect: ConnectConfig = None
     database_to_catalog_mapping: dict[str, str] = None
     default_catalog: str = "ucx_default"
-
-    # Starting path for notebooks and directories crawler
-    workspace_start_path: str = "/"
+    groups: GroupsConfig = None
+    instance_pool_id: str = None
     instance_profile: str = None
+    inventory_database: str = None
+    log_level: str = "INFO"
+    num_threads: int = 10
+    override_clusters: dict[Any, Any] = None
     spark_conf: dict[str, str] = None
+    warehouse_id: str = None
+    workspace_start_path: str = "/"  # Starting path for notebooks and directories crawler
```

Comment:
Please don't reorder the fields in this pr - it's hard to review what is the change

Reply:
Will keep that in mind for the future. I dealing with a tricky mess at the time where adding one more property was causing an error.